ext/dcos-installer/dcos_installer/backend.py

```diff
@@ -36,6 +39,22 @@ def do_configure(config_path=CONFIG_PATH):
         return 0
 
 
+def do_aws_cf_configure():
+    """Tries to generate AWS templates using a custom config.yaml"""
+
+    # TODO(lingmann): Exception handling
+    yaml_config = yaml.load(open(CONFIG_PATH, 'r'))
```

Comment:
you can pass `do_aws_cf_configure(config_path=CONFIG_PATH)` then:

```
config = DCOSConfig(config_path=config_path)
``` 

and this will load the YAML file for you.